bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/internal/i18n/I18nProviderImpl.java

```diff
@@ -153,13 +158,17 @@ private void setLocation(final String location) {
         }
     }
 
-    private void setTimeZone(final ZoneId timeZone) {
-        if (timeZone != null) {
+    private void setTimeZone(final String zoneId) {
+        if (StringUtils.isBlank(zoneId)) {
+            timeZone = TimeZone.getDefault().toZoneId();
+            logger.warn("No time zone set, fallback to the system time zone '{}'.", timeZone.toString());
+        } else {
             try {
-                this.timeZone = TimeZone.getTimeZone(timeZone).toZoneId();
-            } catch (IllegalArgumentException e) {
-                this.timeZone = TimeZone.getDefault().toZoneId();
-                logger.warn("Could not set a new time zone, the system's default time zone will be used ", timeZone, e.getMessage());
+                timeZone = ZoneId.of(zoneId);
+            } catch (DateTimeException e) {
+                timeZone = TimeZone.getDefault().toZoneId();
+                logger.warn("Error setting time zone '{}', fallback to the system time zone '{}': {}", zoneId,
```

Comment:
better: "falling back to..."

Reply:
damn, that was my first attempt but I changed it to be in line with the other logging messages...

Reply:
you could also have fixed the others to be a proper english ;-)